oregon/templates/site.html

```diff
@@ -37,7 +37,12 @@
     #myCarousel {
         margin-top: 10px;
     }
-
+    .place-label {
+    fill:#8C001A;
+    font-size:14;
+    font-weight: bold;
+    text-decoration:none !important;
+    }
     ul.newspaper_pages li.span4 {
```

Comment:
And can we get `:focus` and `:hover` state added?  There are so many links it's hard to tell which one has mouse/keyboard focus.